astropy/wcs/wcs.py

```diff
@@ -33,6 +33,7 @@
 import io
 import itertools
 import os
+import operator
```

Comment:
I think you may not need `operator` any more since you followed my recommendation!

Reply:
Ops, good catch!